tests/console/yast2_dnsserver_service_abort_keep.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
```

Comment:
as we can rename `yast2_dnsserver` to something more concrete, we can just have for the rest: `yast2_dns_server_*`.